package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "github-notifier-chrome",
+  "version": "2.0.0",
+  "description": "> Chrome/Opera extension - Displays your GitHub notifications unread count",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint extension/*.js",
```

Comment:
Why not lint `test/*.js` as well, while there? I know it doesn't matter, but it should help keep the repo clean overall.